src/docker/build/docker-image/Dockerfile

```
@@ -44,28 +43,61 @@ RUN pyinstaller /src/scan_fs.py \
 
 LABEL maintainer="nitrobass24" \
-      description="SeedSync - Seedbox file synchronization tool" \
-      version="0.9.0"
+      description="SeedSync - Seedbox file synchronization tool"
+
+# Copy stripped Python runtime from python-deps stage
+COPY --from=python-deps /usr/local/ /usr/local/
+RUN ldconfig
 
 # Enable non-free repo for full unrar (unrar-free only supports RAR1-3)
 RUN sed -i 's/Components: main/Components: main non-free/' /etc/apt/sources.list.d/debian.sources
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)